README.md

```diff
@@ -135,3 +135,12 @@ We currently use [clippy](https://github.com/rust-lang-nursery/rust-clippy) to e
 This check is run automatically during CI builds, and in a `pre-commit`
 hook. You can run do a clippy check with `./scripts/run_clippy.sh`.
 
+## Development cluster
+
+To spin multiple nodes, you must first spin up one node and then use it as boot node for all the rest:
```

Comment:
They key pair network uses is now persistent between runs (it's stored in the `storage/secret` by default). Renaming --test-node-index is also good, renaming it to --test-network-key-seed

Reply:
persistent between runs is not acceptable for the use case in which we want to automate spinning up multiple nodes.

i used index because it may be clearer that you can use integer values, but we can change to seed and just make it clear in the help text.